### DISCUSSION TITLE
Share link email

### DISCUSSION THREAD

**Discussion by gthb96** [nextcloud/all-in-one#5579] [Questions] [answered]
I have an issue since day one with sharing links via email and I really don't know what I'm doing wrong.

All I did was setting up SMTP in the Adminsettings of NC (nexctcloud@mydomain.xyz), test mails are being sent, password reset mails are working as well.

Only when creating link of shared file, the mail never gets sent to the recipient.
No obvious error messages or anything that would indicate failure.

What am I missing or what could I check?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com. Thanks!

**Reply by gthb96** [reply]
I was not sure if this could be related to AiO :)
